code_generator/GdkCodeGenerator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -116,8 +118,9 @@ namespace <#= qualifiedNamespace #>
                 if (!view.TryGetEntity(op.EntityId.Id, out entity))
                 {
                     LogDispatcher.HandleLog(LogType.Error, new LogEvent("Entity not found during OnAddComponent.")
-                        .WithField(IdType.EntityId.ToString(), op.EntityId.Id)
-                        .WithField("OpName", op.GetType().Name));
+                        .WithField(LoggingUtils.LoggerName, LoggerName)
```

Comment:
Possibly controversial, but how about adding an extension method `WithLoggerName` to `LoggingUtils` to reduce boilerplate a bit?

Reply:
Same for a `WithEntityId` method for below.

Reply:
Fair enough.

Reply:
We did talk about adding the helpers for those specifically though. I think that is not controversial.